src/Configuration/ConfigurationInterface.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the FOSHttpCacheBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\HttpCacheBundle\Configuration;
+
+/**
+ * ConfigurationInterface.
+ *
+ * @author Fabien Potencier <fabien@symfony.com>
+ */
+interface ConfigurationInterface
+{
+    /**
+     * Returns the alias name for an annotated configuration.
+     *
+     * @return string
+     */
+    public function getAliasName();
```

Comment:
```suggestion
    public function getAliasName(): string;
```

we now have modern php, so we can use return type declarations. (and remove the redundant `@return string` annotation)